packages/EasyStandard/src/Sniffs/Classes/AvoidPrivateConstructorSniff.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyStandard\Sniffs\Classes;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+
+final class AvoidPrivateConstructorSniff implements Sniff
+{
+    public function process(File $phpcsFile, $stackPtr)
```

Comment:
- PHPStan will complain about `$stackPtr` not being defined and we can't actually type hint it so the solution here is to add a PHPDoc bloc for it.
- Also we can add the return type hint `: void`